### PR TITLE
Fixes cost values of RCDs

### PR DESCRIPTION
**Pull request by iwishforducks**
## About The Pull Request

fixes cost values to be 1:1 of what the construction would have costed in manual construction for the RCD. also changes all decaseconds to seconds in code for clarity.

the biggest change is windows, they used to cost more than they needed for manual construction:

grille - 4mu - 1 iron sheet - 100% efficiency
directional normal window - 6mu - 1 glass sheet - 66% efficiency
directional reinforced window - 9mu - 1 reinforced glass sheet - 66% efficiency
fulltile normal window - 12mu - 2 glass sheets - 66% efficiency
fulltile reinforced window - 15mu - 2 reinforced glass sheets - 80% efficiency

this PR fixes all of these to be 100% efficient by lowering their matter costs, among some other items like racks or reflector frames.

## Why It's Good For The Game

consistency for material costs is good. most of these incorrect material values are also for things that don't matter, like racks or reflector frames. also decaseconds are gross looking in code

## Changelog

:cl:
fix: Some RCD constructs took more material than manual construction. The RCD cost should be consistent in comparison to manual construction now.
/:cl:

### CODE REVIEW

**Comment by Inari-Whitebear** [comment]
Isn't the point that
RCD = quicker, but costs more
manual = slower, but less cost

Besides, as far as I'm aware, if you create RCD cartridges with ungraded lathe you already compensate for the increased RCD cost (or even have it be cheaper with RCD)?

**Comment by iwishforducks** [comment]
> Isn't the point that RCD = quicker, but costs more manual = slower, but less cost
> 
> Besides, as far as I'm aware, if you create RCD cartridges with ungraded lathe you already compensate for the increased RCD cost (or even have it be cheaper with RCD)?

i think in theory i agree, but i think the point of the RCD at this point is to be a tool that makes construction easier, rather than being some kind of trade off.

things like walls, floors, and airlocks were already at 100% efficiency.

i didn't realize upgraded lathes made matter cartridges cheaper. that seems like it should be fixed but i really don't care about that at all, personally.

**Comment by Inari-Whitebear** [comment]
Hm interesting, I thought RCD was only same price as a wall if the wall already was there and you're repairing or so

File: code/game/turfs/open/space/space.dm [hunk 1]
```diff
@@ -215,9 +215,9 @@ GLOBAL_VAR_INIT(starlight_color, COLOR_STARLIGHT)
 		if(RCD_CATWALK)
 			var/obj/structure/lattice/lattice = locate(/obj/structure/lattice, src)
 			if(lattice)
-				return list("mode" = RCD_CATWALK, "delay" = 0, "cost" = 1)
-			else
 				return list("mode" = RCD_CATWALK, "delay" = 0, "cost" = 2)
+			else
+				return list("mode" = RCD_CATWALK, "delay" = 0, "cost" = 4)
```
Comment (Fikou) [inline code/game/turfs/open/space/space.dm:220]: why the changes they dont rly make sense

Comment (iwishforducks) [reply code/game/turfs/open/space/space.dm:220]: talked to fikou about this over discord:

catwalks are built using iron rods on lattice.

lattice takes 1 iron rod, which is 2 matter, and the catwalk takes 1 iron rod, which is also 2 matter. totals to 4 matter for the full build.

**Comment by CRITAWAKETS** [comment]
I think this is actually intended, but honestly it's kind of a bummer and makes the RCD deceptively terrible at making anything that's not reconstruction since you'll be spending a LOT of materials for basically anything.
